donate/utility/acoustic/acoustic.py

```diff
@@ -137,13 +137,14 @@ def send_mail(self, to, campaign_id, fields=None, bcc=None, save_to_db=False):
                 # explicitly block, just in case the implicit behaviour changes in the future
                 schedule.run(blocking=True)
                 break
-            except ConnectionError:
+            except (ConnectionError, ReadTimeout) as err:
                 if attempt != 2:
-                    logger.error("Error connecting to Acoustic to send email receipt. Trying again.")
+                    logger.error("Error connecting to Acoustic while sending email receipt. Trying again.")
                 else:
                     logger.error(
-                        f"Could not send email receipt. Unable to connect to Acoustic {send_retries} retries."
+                        f"Could not send email receipt. Unable to connect to Acoustic after {send_retries} retries."
                     )
+                    logger.error(err)
```

Comment:
Per ticket request, if we are not able to send the email after x number of tries, we log an error message as well as the actual exception for further detail!